package.json

```diff
@@ -39,15 +39,15 @@
     "fibers": "^1.0.14",
     "font-awesome": "^4.6.3",
     "griddle-react": "^0.6.1",
-    "i18next": "^3.4.3",
+    "i18next": "^3.5.2",
```

Comment:
@andela-mmakinde Any reason why you changed the versions of some of the dependencies?